cephfs/file_ops.go

```diff
@@ -0,0 +1,34 @@
+//go:build !nautilus && ceph_preview
+// +build !nautilus,ceph_preview
+
+package cephfs
+
+/*
+#cgo LDFLAGS: -lcephfs
+#cgo CPPFLAGS: -D_FILE_OFFSET_BITS=64
+#include <stdlib.h>
+#include <cephfs/libcephfs.h>
+*/
+import "C"
+
+import (
+	"unsafe"
+)
+
+// MkNod creates a regular, block or character special file.
+//
+// Implements:
+//
+//	int ceph_mknod(struct ceph_mount_info *cmount, const char *path, mode_t mode,
+//				   dev_t rdev);
+func (mount *MountInfo) MkNod(path string, mode uint16, dev uint16) error {
```

Comment:
Can you spell it Mknod to better match https://pkg.go.dev/golang.org/x/sys/unix#Mknod, please?

Reply:
Done.